package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3017,28 +3019,25 @@ def dimensions(self, box):
         """
         self.universe.dimensions = box
 
-    @property
-    def ts(self):
-        """Temporary Timestep that contains the selection coordinates.
+    def get_partial_ts(self):
```

Comment:
I'd just remove the caching here.  I can't think of a case where you'd want to retrieve the `.ts` multiple times to use